### ISSUE TITLE
Unclickable 1px area above all panel elements [touchpad only]

### ISSUE THREAD

**Issue by cyber-doug** [solus-project/budgie-desktop#519]
This affects usability and it is very very much frustrating.

Using the touchpad, I can't hit any icons from budgie panel when clicking at the **very top** of each of them. I cannot reproduce the issue while using a mouse.

There is a 1px area where the touchpad does not work properly. It is exactly where tasklist indicators appear, and also affects notification area, budgie-menu and everything else intended to be clickable at the very top of the screen, up to the last pixel. Instead, it keeps switching highlighting on and off while I move the cursor inside the same icon and in that 1px area. If the icon is highlighted, it works, but it won't highlight consistently while I hover them at that 1px area.

This behaviour is difficult to describe, because sometimes the cursor finds the right spot where it's clickable. Perhaps it's related to Synaptics? Moving the panel to the bottom does not solve it, it will only happen at the bottom, in the same way. **1px unstable area when using the touchpad**.

This issue affects me since Solus 1.1, and it hasn't changed or improved.
The screenshot utility does not show the mouse pointer in the picture despite of having the respective option selected (probably another bug). I need help to collect information that will help debugging this.

**Comment by ikeydoherty** [comment]
Its actual padding in the applet, which has since been removed in git, when we introduced the 1:1.1 aspect ratio to remove margin/padding hacks, which basically means the click height if straight is as expected.

Budgie Menu has since had been turned into a GtkToggleButton - with padding removed, allowing clicking right into the corner.

As for notification area, it depends which you're talking about. If you actually mean X11 tray applets, they cannot be clicked at the top because those are _foreign windows_, they process their own clicking. i.e. precisely square areas.

Note that this includes things like nm-applet, or hexchat, for example. In terms of the Budgie-specific-craqbuggery, that's all fixed in git. I look forward to you getting a more 1px friendly Budgie. :)
